db.js

```diff
@@ -168,3 +175,27 @@ export const populateUserCache = async (cache = userCache) => {
 	}
 };
 
+
+
+export const insertTimezone = async ( _id, timezone = null , dbConnection = connectDB) => {
+	try {
+		const db = await dbConnection();
```

Comment:
Just curious here, why pass in dbConection and default it to connectDB? Why not just call connectDB further down because it's available to this function?

Is there an instance where the this method may need to connect do a different db?

@Claudi0-V

Reply:
this is dependency injection. one of the cool things is that allows us to pass a fake db when testing.

it also makes the code more clear, you can more easily find out which parts are from inside the function itself